core/services/fluxmonitorv2/integrations_test.go

```diff
@@ -464,7 +464,7 @@ func TestFluxMonitor_Deviation(t *testing.T) {
 
 			type v struct {
 				count     int
-				updatedAt string
+				updatedAt int64
```

Comment:
Using a string representation of the number caused testify to use lexicographical comparison, so that e.g. `100 < 60`